scripts/Muon/GUI/Common/fitting_tab_widget/fitting_tab_model.py

```diff
@@ -463,9 +467,9 @@ def create_hashable_fit_workspace_names(self):
         return equiv_workspace_list
 
     def update_ws_fit_function_parameters(self, workspace, params):
-        workspace_hash = self.create_equivalent_workspace_name(workspace)
+        eq_workspace_name = self.create_equivalent_workspace_name(workspace)
```

Comment:
what does eq stand for? I would assume equation

Reply:
Equivalent, admittedly it’s not fully clear, but in this case the return name is deducible from the function name

Reply:
could we have equiv like a few lines below? That was clearer